### PR TITLE
OCPBUGS-28230: enforce termination message policy on all platform pods

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: assets/controller.yaml [hunk 1]
```diff
@@ -226,6 +232,7 @@ spec:
             requests:
               memory: 20Mi
               cpu: 10m
+          terminationMessagePolicy: FallbackToLogsOnError
           volumeMounts:
           - mountPath: /etc/tls/private
             name: metrics-serving-cert
```
[the remark this answers is not in the document]
Comment (deads2k) [reply assets/controller.yaml:238]: > Any reason not to change the csi-liveness-probe container too?

Just missed it.  Good catched.  Added and re-pushed.

**Comment by bertinatto** [comment]
/approve
/label docs-approved
/label px-approved
Not a user-facing change.

**Pull request merged**